module/data.js

```diff
@@ -0,0 +1,33 @@
+const speaker = [
+
+  {
+    id: 'speaker1',
+    image: './images/four.jpg',
+    name: 'Herbart obi',
+    profession: 'Executive director of the Wikimedia Foundation',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker2',
+    image: './images/IMG_2508.jpg',
+    name: 'Bright Eze',
+    profession: 'Berkman Professor of Enterpreneurial',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker3',
+    image: './images/two.jpg',
+    name: 'Alima Buki',
+    profession: 'Executive director of the Wikimedia Foundation',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+  {
+    id: 'speaker4',
+    image: './images/IMG_6177 copy.jpg',
+    name: 'Progress Eze',
+    profession: 'Berkman Professor of Enterpreneurial',
+    aboutMe: 'Benkler studies commons-based peer production.',
+  },
+];
+
```

Comment:
- [ ] Good job implementing all the information of speakers in an array of objects with all the needed info. But I think it would be better if you add other speakers, and after that improve the design to match the one from the template on both **Mobile** and Desktop versions. Right now, I can see that each speaker picture misses the grid background image, the horizontal line below the speaker name,...

### Template Design
![Screenshot from 2023-01-06 16-03-29](https://user-images.githubusercontent.com/83042742/212492443-ebd24370-11d8-4cff-b16c-cdab51805ee0.png)

### Your  Design
![Screenshot from 2023-01-14 21-23-03](https://user-images.githubusercontent.com/83042742/212492552-c3d1578c-00e2-4099-8d25-bd6fdd11cf67.png)

Reply:
- [ ] [REQUIRED ] As mentioned by the previous reviewer. According to the project requirement "speakers"  section should be created dynamically in JavaScript. For that 
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

```
{
  name: "Specaker name ", 
  role: "Java developer facebook", 
  description: "Speaker description ...", 
  photo: "image_url_here"
}

```